site/kubernetes/operator/quickstart-operator.md

```diff
@@ -0,0 +1,237 @@
+replicaset.apps/rabbitmq-cluster-operator-54f948d8b6   1         1         1       2m10s
+```
+- a new custom resource `rabbitmqclusters.rabbitmq.com`. The custom resource allows us to define an API for the creation of RabbitMQ Clusters.
+```shell
+kubectl get customresourcedefinitions.apiextensions.k8s.io
+
+NAME                                             CREATED AT
+...
+rabbitmqclusters.rabbitmq.com                    2021-01-14T11:12:26Z
+...
+```
+- and some rbac roles. These are required by the Operator to create, update and delete RabbitMQ Clusters.
+
+## Hello RabbitMQ!
+
+Now that we have the Operator deployed, let's create the simplest RabbitMQ Cluster.
+
+This example can be found in the [Cluster Operator GitHub repo](https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/hello-world).  As mentioned on the page:
+> This is the simplest `RabbitmqCluster` definition. The only explicitly specified property is the name of the cluster. Everything else will be configured according to the Cluster Operator's defaults.
+
+The [examples folder](https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/) has many other references such as creating a RabbitMQ Cluster with TLS, mTLS, setting up a Cluster with production defaults, adding community plugins, etc.
+
+Continuing with our example, we will submit the following yaml to Kubernetes:
+
+```yaml
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+	name: hello-world
+```
+Submit this using the following command -
+```shell
+kubectl apply -f https://raw.githubusercontent.com/rabbitmq/cluster-operator/main/docs/examples/hello-world/rabbitmq.yaml
+```
+
+This will create  a RabbitMQ cluster called `hello-world` in the current namespace. You can see the RabbitMQ Cluster as it is being created:
+
+```shell
+watch kubectl get all
+
+NAME                       READY   STATUS    RESTARTS   AGE
+pod/hello-world-server-0   1/1     Running   0          2m
+
+NAME                        TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)              AGE
+service/hello-world         ClusterIP   10.75.242.149   <none>        5672/TCP,15672/TCP   2m
+service/hello-world-nodes   ClusterIP   None            <none>        4369/TCP,25672/TCP   2m
+service/kubernetes          ClusterIP   10.75.240.1     <none>        443/TCP              4h1m
+
+NAME                                  READY   AGE
+statefulset.apps/hello-world-server   1/1     2m
+```
+
+You will also be able to see an instance of the `rabbitmqclusters.rabbitmq.com` custom resource created.
+
+```shell
+kubectl get rabbitmqclusters.rabbitmq.com
+
+NAME          AGE    STATUS
+hello-world   4m1s
+```
+
+You may also use the kubectl rabbitmq plugin to list the RabbitMQ Clusters deployed -
+
+```shell
+kubectl rabbitmq list
+NAME          AGE    STATUS
+hello-world   4m10s
+```
+
+Briefly breaking down what the `hello-world` RabbitMQ Cluster has created in terms of Kubernetes objects -
```

Comment:
I wonder this section on child resources could be too much details for a `quick start`. I think we talk about child resources in other docs as well, so maybe we can skip on the information here to keep quick start simple and short.

Reply:
I read the document again and agree with you. I think it's better to have a dedicated document that focuses on this than to cover it briefly here. Will trim it.